library/src/main/java/uk/co/samuelwall/materialtaptargetprompt/MaterialTapTargetPrompt.java

```diff
@@ -902,6 +903,7 @@ protected void onPromptTouched(final MotionEvent event, final boolean targetTapp
         private boolean mHasIconDrawableTint;
         private int mIconDrawableColourFilter;
         private View mTargetRenderView;
+        private boolean mIdleAnimationEnabled;
```

Comment:
This needs to be set to `true` so that the idle animation is enabled by default.

```Java
private boolean mIdleAnimationEnabled = true;
```